sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -55,7 +55,8 @@ dependencies {
   shadow library.java.joda_time
   shadow library.java.google_cloud_core
   shadow library.java.google_cloud_spanner
-  shadow library.java.bigtable_protos
+  shadow library.java.proto-google-cloud-bigtable-v2
```

Comment:
I don't think you want to use `-` as it is not a valid symbol in variable names, please stick with `_`